src/coprocessor/metrics.rs

```diff
@@ -18,7 +18,9 @@ use crate::{
 make_auto_flush_static_metric! {
     pub label_enum ReqTag {
         select,
+        select_by_range_cache,
         index,
+        index_by_range_cache,
```

Comment:
How about adding another label for engine types?

For example:
```
some_metric{req=select, engine=rocksdb}
some_metric{req=index, engine=in_memory_engine}
```

Reply:
We may not know the exact engine type at coprocess layer. Even for range cache engine here, we know from snashot that it is a engine implementing RangeCacheEngine rather than we know it is IME.